appengine/cloudsql/server.js

```diff
@@ -89,37 +107,40 @@ app.get('/', (req, res, next) => {
     userIp: crypto.createHash('sha256').update(req.ip).digest('hex').substr(0, 7)
   };
 
-  insertVisit(visit, (err, results) => {
-    if (err) {
-      next(err);
-      return;
-    }
-
-    // Query the last 10 visits from the database.
-    getVisits((err, visits) => {
-      if (err) {
-        next(err);
-        return;
-      }
-
+  insertVisit(visit)
+    .then(() => {
+      // Query the last 10 visits from the database.
+      return getVisits();
+    })
+    .then((visits) => {
       res
         .status(200)
         .set('Content-Type', 'text/plain')
         .send(`Last 10 visits:\n${visits.join('\n')}`)
         .end();
+    })
+    .catch((err) => {
+      next(err);
     });
-  });
 });
 
-if (module === require.main) {
-  // [START listen]
-  const PORT = process.env.PORT || 8080;
-  app.listen(PORT, () => {
-    console.log(`App listening on port ${PORT}`);
-    console.log('Press Ctrl+C to quit.');
-  });
-  // [END listen]
+// Get type of SQL client to use
+const sqlClient = process.env.SQL_CLIENT;
+if (sqlClient === 'pg') {
```

Comment:
No need for conditional if we rely on passing `process.env.SQL_CLIENT` to the Knex constructor